desktop/analytics/account_creation.js

```diff
@@ -69,6 +69,18 @@ if (Cookies.get('analytics-signup')) {
   }
 }
 
+// Created account on the Inquire via Phone modal (via email)
+analyticsHooks.on('mediator:auth:sign_up:success', ({context, user}) => {
```

Comment:
is this using `analyticsHooks` or `mediator` events? Maybe it's better to do `analyticsHooks.trigger` from the view really explicitly, to encapsulate its use. It also could then just be `analyticsHooks.trigger('signUpFromPhoneModal')` or something even more descriptive than this mediator thing.